config.json

```diff
@@ -25,6 +26,14 @@
     "eks",
     "kubectl"
   ],
+  "settings": [
```

Comment:
Are you reverting the namespace settings then, or is it a false github diff?

Reply:
as discussed namespaces remain in settings even though the plugin makes no use of them.